.pre-commit-config.yaml

```diff
@@ -19,7 +19,7 @@ repos:
       - id: black
         files: '(src|tests).*py'
   - repo: https://github.com/PyCQA/isort
-    rev: 5.6.4
+    rev: 5.12.0
```

Comment:
this was bumped to fix:
>       RuntimeError: The Poetry configuration is invalid:
        - [extras.pipfile_deprecated_finder.2] 'pip-shims<=0.3.4' does not match '^[a-zA-Z-_.0-9]+$'

for more info see https://levelup.gitconnected.com/fix-runtimeerror-poetry-isort-5db7c67b60ff